GTViewController/Classes/Nav Classes/GTViewController.m

```diff
@@ -1190,6 +1193,16 @@ - (void)dismissFollowupModal {
     
 }
 
+
+- (NSString *)currentPackageCode {
```

Comment:
Please use property statements to generate your setters and getters unless you really need to do something custom